CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.8 (not released yet)
 
+- [#763] Introduce CASC_VAULT_PATHS to allow multiple vault paths to read from. CASC_VAULT_PATH kept for backwards compatibility and offering multi path too now.
```

Comment:
Linking to #763 you have to give the full address in the changelog like below :cry:

Reply:
True. Codacy also caught it :)